### PR TITLE
Netflixossbuild

### PR DESCRIPTION
**Pull request by rspieldenner**

### CODE REVIEW

**Comment by pebo** [comment]
Could this one be merged? It also seems to include fix https://github.com/Netflix/SimianArmy/pull/148

File: build.gradle [hunk 1]
```diff
@@ -1,58 +1,40 @@
...
-}
-
-allprojects {
-    repositories { 
-        mavenLocal()
-        mavenCentral() // maven { url: 'http://jcenter.bintray.com' }
-    }
+repositories { 
+    mavenLocal()
+    jcenter()
```
Comment (gorzell) [inline build.gradle]: Why the switch to jcenter?

**Comment by gorzell** [comment]
One minor question, otherwise looks good.

**Comment by gorzell** [comment]
Also needs to be updated before I can merge it.

**Pull request merged**